Terasology/LightAndShadowResources
# Health bar skin files organization

Related PR
* https://github.com/Terasology/LightAndShadow/pull/124

# Result

We now have 1 skin file (`healthHud.skin`) to change the size of the icons within one file, which is located within the LightAndShadow module. The rest 3 skin files located in the LightAndShadowResources module inherit the size from the mentioned skin file.

## Changes

* Deleted all size attributes within the 3 skin files within LightAndShadowResources
* Made them inherit the size within the LightAndShadow skin about the health bar.

The file from which they inherit from can be found through this draft PR: https://github.com/Terasology/LightAndShadow/pull/124.

## How to test

Make sure you have set up the https://github.com/Terasology/LightAndShadow/pull/124 as well.

* Start a LaS game (both old or new game file works fine)
* Check the size of the health bar
* You can enter a team by clicking one of the two (black or red) boxes, to check the size of the other teams' icon size.

Extra: you can go ahead and change the size number placed within the `healthHud.skin` from the lines:
```json
"icon": {
    "fixed-width": 30,
    "fixed-height": 30
}
```
to make extra sure.
